scripts/extract-release-notes.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Nice! I wonder if the script should also validate that the version number is right. This would prevent us from releasing CCF version `N` with the wrong release notes if CCF version `N-1` is the latest documented release. 

We've already got the `cmake` machinery for this (the CCF version number is derived from the git tag, see `version.cmake`) so we could just add a check for this too?

Reply:
Sounds good, yes.